torch/csrc/api/src/nn/modules/activation.cpp

```diff
@@ -475,8 +480,8 @@ void MultiheadAttentionImpl::reset() {
       options.embed_dim()).bias(options.bias()))
   );
   if (options.add_bias_kv()) {
-    bias_k = torch::empty({1, 1, options.embed_dim()});
-    bias_v = torch::empty({1, 1, options.embed_dim()});
+    bias_k = register_parameter("bias_k", torch::empty({1, 1, options.embed_dim()}));
+    bias_v = register_parameter("bias_v", torch::empty({1, 1, options.embed_dim()}));
   } else {
     bias_k = {};
```

Comment:
Do we need to `register_parameter('bias_k', {}, /*requires_grad=*/false)`?

Reply:
Based on python impl, No. It only register the non-empty case for bias_k and bias_v.